packages/@n8n/task-runner/src/js-task-runner/built-ins-parser/built-ins-parser.ts

```diff
@@ -131,6 +131,9 @@ export class BuiltInsParser {
 			state.markExecutionAsNeeded();
 		} else if (node.name === '$prevNode') {
 			state.markPrevNodeAsNeeded();
+		} else if (node.name === 'items' || node.name === 'item') {
+			// item is deprecated but we still need to support it
+			state.markInputAsNeeded();
```

Comment:
This check could also be made part of the `$input` and `$json` check so all identifiers relating to input are in a single spot.

Reply:
Done!